src/routes/users.route.ts

```diff
@@ -0,0 +1,11 @@
+import {createUser, getUsers, getUser, deleteUser, updateUser} from "../controllers/users.controller";
+import {FastifyInstance, FastifyServerOptions} from "fastify";
+
+export async function routes (fastify: FastifyInstance, options: FastifyServerOptions) {
+    fastify.post('/create', createUser)
+    fastify.get('/', getUsers)
+    fastify.get('/:userId', getUser)
+    fastify.delete("/:userId", deleteUser)
+    fastify.put("/update/:userId", updateUser)

```

Comment:
here is the same, remove update, just left      fastify.put("/:userId", updateUser)